### DISCUSSION TITLE
Support for Alta HR?

### DISCUSSION THREAD

**Discussion by TypicalZedF** [200Tigersbloxed/HRtoVRChat_OSC#8] [Ideas]
Could we get support for an old Alta HR FitBit? It's an old device that some people most likely have in their closets. Would be nice to have support for it.

**Comment by 200Tigersbloxed** [comment]
Hi,
Fitbit's SDK only has support for Gallery Supported devices. There is no way, at least to my knowledge, of being able to install/sideload an app onto non-Gallery watches. The only possible 'solution' for this is to read raw bluetooth messages from the Fitbit device itself, then translate that into a HeartRate. You can then pass this data through to a textfile, or an [SDK](https://github.com/200Tigersbloxed/HRtoVRChat_OSC/blob/main/SDK.md). 

Sorry about that, let me know if there's any more questions!

For a list of Gallery Devices, see [here](https://github.com/200Tigersbloxed/FitbitHRtoWS#support)

**Reply by TypicalZedF** [reply]
Actually, I ended up finding someone who is willing to make something that works with this. Apparently, they can use the FitBit web API. The beneficial result is it should work with ALL Fitbit devices, and won't require an app downloaded on the watch.

**Reply by 200Tigersbloxed** [reply]
Sweet! Please let me know how this goes! I've been looking for people to add their devices with an SDK to feature! (also I'd love to try it out on my old Charge HR)
